.github/workflows/ci.yml

```diff
@@ -274,4 +269,9 @@ jobs:
         run: ctest --output-on-failure --no-tests=error -C Release -j 4
 
       - name: Benchmarks
+        if: ${{ !contains(matrix.os, 'windows') }}
         run: ./build/benchmarks/bark_benchmarks --benchmark_display_aggregates_only=true
+
+      - name: Benchmarks
+        if: ${{ contains(matrix.os, 'windows') }}
+        run: ./build/benchmarks/Release/bark_benchmarks --benchmark_display_aggregates_only=true
```

Comment:
Perhaps you can make cmake output the build folder. Otherwise you can also call cmake --install, to get it to a specific folder. Would also be nice in terms of testing

Reply:
I will leave this for a follow up.